config.sh

```diff
@@ -17,6 +18,21 @@ LIBWEBP_VERSION=1.1.0
 BZIP2_VERSION=1.0.8
 LIBXCB_VERSION=1.14
 
+# workaround for multibuild bug with .tar.xz
+function untar {
+    local in_fname=$1
+    if [ -z "$in_fname" ];then echo "in_fname not defined"; exit 1; fi
+    local extension=${in_fname##*.}
+    case $extension in
+        tar) tar -xf $in_fname ;;
+        gz|tgz) tar -zxf $in_fname ;;
+        bz2) tar -jxf $in_fname ;;
+        zip) unzip -qq $in_fname ;;
+        xz) unxz -c $in_fname | tar -xf - ;;
```

Comment:
This line is wrong in the multibuild version causing an error in `tar`; will report tomorrow.